examples/blank-gpu/.saturn/saturn.json

```
@@ -0,0 +1,13 @@
+{
+    "image": "saturncloud/saturn-gpu:2021.01.13",
```

We shouldn't recommend that people use saturn-gpu anymore. It's never going to get anymore updates.

```suggestion
    "image": "saturncloud/saturn-rapids:2021.02.22",
```